postgres/tests/test_statements.py

```diff
@@ -627,6 +627,81 @@ def obfuscate_sql(query, options=None):
     assert metric['dd_commands'] == expected_metadata_payload['commands']
 
 
+@pytest.mark.parametrize("pg_stat_statements_view", ["pg_stat_statements", "datadog.pg_stat_statements()"])
```

Comment:
Are these parameters relevant for the `test_statement_reported_hostname` test?

Reply:
Like you pointed out, we can simplify the tests, so this is not necessary.